mesh/node.c

```diff
@@ -2267,14 +2311,17 @@ static void setup_node_interface(struct l_dbus_interface *iface)
 	l_dbus_interface_method(iface, "VendorPublish", 0, vendor_publish_call,
 						"", "oqqay", "element_path",
 						"vendor", "model_id", "data");
-
+	l_dbus_interface_method(iface, "SetSeqNr", 0, set_seq_nr_call, "",
```

Comment:
I would rename it to `UpdateSequenceNumber`, it's not supposed to be a straightforward setter.